testsuite/testcases/src/lib.rs

```diff
@@ -31,8 +31,8 @@ use rand::{rngs::StdRng, SeedableRng};
 use std::time::{Duration, Instant, SystemTime, UNIX_EPOCH};
 use tokio::runtime::Builder;
 
-const WARMUP_DURATION_FRACTION: f32 = 0.05;
-const COOLDOWN_DURATION_FRACTION: f32 = 0.05;
+const WARMUP_DURATION_FRACTION: f32 = 0.07;
```

Comment:
Any particular reason for this change?

Reply:
yeah, warmup is more important, especially at high loads. with this I am adding a 40% bigger warmup, without affecting much total or cooldown.

LoadVsPerf uses much bigger warmup, and that is useful. I might change separately so that this is configurable , but not in this PR